### DISCUSSION TITLE
Domain will Expire email

### DISCUSSION THREAD

**Discussion by lorensr** [vercel/community#2247] [Bug Report]
I received the below email:

![image](https://user-images.githubusercontent.com/251288/235383809-7ca6bf37-f72f-4ef5-bd96-321f4a543ebb.png)

which linked to https://vercel.com/docs/concepts/projects/domains#renewing-domains?utm_medium=email&utm_source=notifications

3 requests:

- fix double comma
- add link to my domain page so I can click auto renew
- add link to page where I can renew for N more years

**Comment by amyegan** [comment]
Thanks for sharing this, @lorensr! I'll let the team know so we can get the email template updated

**Comment by amyegan** [comment]
Thanks again for reporting this. The template was updated with more useful links, and the extra comma was removed 😄

**Comment by lorensr** [comment]
Nice, thanks! ☺️
